package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "jobs-ejs",
+  "version": "1.0.0",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon app.js"
```

Comment:
The assignment asks you to define both the start and dev scripts. The start script is the way your app is executed in a production environment so it should use `node` instead of `nodemon` (which will be used by the dev script).